src/types/statement.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+export type SimpleStatement = {
```

Comment:
I know that feels super heavyweight but we should generate these from the protobufs, probably